lib/media/preload_manager.js

```diff
@@ -363,6 +363,9 @@ shaka.media.PreloadManager = class extends shaka.util.FakeEventTarget {
   attachManifest(manifest, parser, parserFactory) {
     this.manifest_ = manifest;
     this.parser_ = parser;
+    if (this.parser_ && this.parser_.setMediaElement) {
+      this.parser_.setMediaElement(/* mediaElement= */ null);
```

Comment:
Is this to un-assign the media element, in the case of `unloadAndSavePreload` or `detachAndSavePreload`?

If so, it might be better to put this code into `shaka.Player.savePreload_` instead, for clarity.

Reply:
Done!